### ISSUE TITLE
Player inventory editor

### ISSUE THREAD

**Issue by IIPedro** [Multiverse/Multiverse-Inventories#393]
### Feature request

**Feature description**
An inventory editor, sort of like Essentials' /invsee. More specifically, please make it possible to edit offline players' inventories and hotbar, armor slots and offhand too.

**How the feature is useful**
Out of many reasons, in my case I'd like to have that feature to better combat agains griefers and scammers, not to mention hackers with illegal items.

Thanks, and have a nice day!

**Comment by benwoo1110** [comment]
So you are looking for something like open inv? https://dev.bukkit.org/projects/openinv

**Comment by IIPedro** [comment]
Yes, however OpenInv does not work well with Multiverse Inventories.

**Comment by northeastprince** [comment]
Love this idea, as I'm pretty sure /invsee from commands like OpenInv and Essentials/EssentialsX only work with the world someone is currently in.
